<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ui_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>680</width>
      <height>680</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="ui_groupbox_1">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Maze Generator</string>
    </property>
    <widget class="QPushButton" name="ui_buttonP1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_buttonP2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ui_groupbox_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>210</y>
      <width>291</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Maze Search</string>
    </property>
    <widget class="QRadioButton" name="ui_buttonR1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Random mouse</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">ui_groupA</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="ui_buttonR2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Wall follower</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">ui_groupA</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="ui_buttonR3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Trémeaux's algorithm</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">ui_groupA</string>
     </attribute>
    </widget>
    <widget class="QPushButton" name="ui_buttonP3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ui_buttonR4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Flood fill</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">ui_groupA</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ui_groupbox_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>460</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Computing time</string>
    </property>
    <widget class="QLineEdit" name="ui_lineedit_1">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_lineedit_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_info_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Time [1 cell forward]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_info_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Nb of [1 cell forward]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_info_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Time [90° rotation]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_info_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Nb of [90° rotation]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_lineedit_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_lineedit_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>1.8</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_info_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Time [full path]:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_lineedit_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="ui_groupA"/>
 </buttongroups>
</ui>
